src/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/IncrementalHash.net46.cs

```diff
@@ -116,7 +114,7 @@ public byte[] GetHashAndReset()
                 _hash.Initialize();
             }
 
-            _hash.TransformFinalBlock(s_empty, 0, 0);
+            _hash.TransformFinalBlock(Array.Empty<byte>(), 0, 0);
```

Comment:
I don't believe we build these against net 4.5 in TFS. For the framework itself we do build against a filtered version of mscorlib (which you created @stephentoub :)) to avoid patch family issues with Array.Empty but I don't think that applies to this code in the context of the corefx builds.